db/queries/cards.sql

```diff
@@ -89,6 +89,25 @@ VALUES ($1, $2)
 ON CONFLICT(user_id) DO UPDATE
 SET params = $2;
 
+-- name: CountCardsInOtherDecks :one
+SELECT COUNT(*)
+FROM wordvault_cards
+WHERE user_id = $1
+    AND lexicon_name = $2
+    AND alphagram = ANY(@alphagrams::text[])
+    AND ((deck_id IS NULL AND sqlc.narg(deck_id)::BIGINT IS NOT NULL)
+        OR (deck_id IS NOT NULL AND deck_id != COALESCE(sqlc.narg(deck_id)::BIGINT, -1)));
```

Comment:
is this [`is distinct from`](https://www.postgresql.org/docs/current/functions-comparison.html)?

Reply:
Oh nice, yeah will use that instead

Reply:
Ended up just using COALESCE and having the SQL layer treat 0 as null -- this code was written before changing the API to use `0` instead of `nil`